neofetch

```diff
@@ -8823,6 +8833,82 @@ EOF
     fi
 }
 
+ssh_connect(){
+    ssh_command="ssh"
+    ssh_opts="-t"
+
+    # filter out first --ssh HOST argument before passing to remote host
+    ssh_arg=no
+    newargs=""
+    while [[ "$1" ]]
+    do
+        if [[ "$ssh_arg" == "no" ]] && [[ "$1" == "--ssh" ]]
+        then
+            shift 2
+            ssh_arg=yes
+            continue
+        fi
+        newargs="$newargs $( printf "%q" "$1" )"
+        shift
+    done
+
+    # generate random filename (with fallbacks)
```

Comment:
This can replace the entire list of tests.

```sh
IFS=$'\n' read -d "" -t 1 rand < /dev/urandom
rand=${rand//[^[:alnum:]]}
rand=n${rand:0:20}
```

Reply:
Fixed using $RANDOM and printf "%x"